ts/Core/Axis/ScrollbarAxis.ts

```diff
@@ -75,6 +75,13 @@ class ScrollbarAxis {
                     this: Highcharts.Scrollbar,
                     e: Highcharts.ScrollbarChangedEventObject
                 ): void {
+                    // #13473 Bars shrink if min is not set while scrolling
+                    if ((this.chart.options.chart?.type === 'column' ||
+                        this.chart.options.chart?.type === 'bar') &&
+                        !defined(axis.options.min)
+                    ) {
+                        axis.options.min = 0;
```

Comment:
Setting `options.something` is not a good idea, from my experience creates more bugs later. Since this callback sets extremes on Axis, perhaps extremes are wrong? As I mentioned above, I don't see `Axis.threshold` used anywhere here, so maybe `axisMin` should consider that value too?

Reply:
Yes I was worried about that as well, I've been trying to come up with a better solution. Thanks, I'll look into `Axis.threshold` and see what happens 😅